plugins/sap_hana_logs.yaml

```diff
@@ -59,8 +60,11 @@ template: |
                 warn: w
                 error: e
                 fatal: f
-
+  extensions:
+    file_storage:
+      directory: /tmp/storage 
```

Comment:
Yeah I missed this at first because it was SAP HANA but this should be a parameter with a default value of `OIQ_OTEL_COLLECTOR_STORAGE`. We do want to give customers the ability to set this without having to change that ENV if they aren't aware of it or don't want to.